inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/InAppMessaging.kt

```diff
@@ -126,18 +133,22 @@ abstract class InAppMessaging internal constructor() {
         internal fun initialize(
             context: Context,
             isCacheHandling: Boolean = false,
+            subscriptionKey: String? = null,
+            configUrl: String? = null,
             configScheduler: ConfigScheduler = ConfigScheduler.instance()
         ) {
             val manifestConfig = InApp.AppManifestConfig(context)
 
             // `manifestConfig.isDebugging()` is used to enable/disable the debug logging of InAppMessaging SDK.
             // Note: All InAppMessaging SDK logs' tags begins with "IAM_".
-            instance = InApp(context, manifestConfig.isDebugging(), isCacheHandling = isCacheHandling)
+            if (instance is NotConfiguredInAppMessaging) {
+                instance = InApp(context, manifestConfig.isDebugging(), isCacheHandling = isCacheHandling)
+            }
 
             Initializer.initializeSdk(
                 context = context,
-                subscriptionKey = manifestConfig.subscriptionKey(),
-                configUrl = manifestConfig.configUrl()
+                subscriptionKey = subscriptionKey ?: manifestConfig.subscriptionKey(),
```

Comment:
We can add checking here that if subkey or config URL is empty (after trim), it will fallback to build config.

Reply:
Added